writeengine/server/we_dmlcommandproc.cpp

```diff
@@ -3121,6 +3121,20 @@ uint8_t WE_DMLCommandProc::processUpdate(messageqcpp::ByteStream& bs,
                             break;
                         }
 
+                        case CalpontSystemCatalog::LONGDOUBLE:
+                        {
+                            double dll = row.getDoubleField(fetchColPos);
```

Comment:
Shouldn't this be long double in the rest of this case statement?

Reply:
blech. Of course.